### PR TITLE
FE-913 Move Global Styles to Styled Components

### PR DESCRIPTION
**Pull request by jonambas**
### What Changed
– Global styles are no longer bundled into a styles.css export
– Global styles are generated through ThemeProvider

### How To Test or Verify
- Run storybook with `npm run start:storybook`
- Verify global styles are working in any story

### PR Checklist
- [x] Add your changes to `unreleased.md` in the root directory. If you plan on publishing immediately after merging (such as a hotfix) or aren't making changes to a published package, you can ignore this step.
- [ ] Provide screenshots or [screen recordings](https://getkap.co/) for any visual changes.
- [x] Get approval from a UX team member (**#uxfe** or **#design-guild** on Slack) for any visual changes.

### CODE REVIEW

File: packages/matchbox/src/index.js [hunk 1]
```diff
@@ -1,2 +1 @@
-import './styles/global.scss';
```
Comment (bkemper) [inline packages/matchbox/src/index.js]: Should we delete this file?

Comment (bkemper) [reply packages/matchbox/src/index.js]: Or, is this saved, so you could use it with `skipGlobalStyles`?

File: packages/matchbox/src/components/ThemeProvider/globalStyles.js [hunk 2]
```diff
@@ -0,0 +1,141 @@
+import { tokens } from '@sparkpost/design-tokens';
+
+export default `
+  html {
+    position: relative;
+    box-sizing: border-box;
+    font-size: 16px;
```
Comment (bkemper) [inline packages/matchbox/src/components/ThemeProvider/globalStyles.js]: Like we were talking about in the change, should we use `$DEFAULT_FONT_SIZE`?

File: stories/storyHelpers/StoryContainer.js [hunk 3]
```diff
@@ -1,5 +1,4 @@
 import React from 'react';
-import '../../packages/matchbox/styles.css';
```
Comment (bkemper) [inline stories/storyHelpers/StoryContainer.js]: 🥇

File: packages/matchbox/src/components/ThemeProvider/globalStyles.js (hunk 2 above)
Comment (jonambas) [reply packages/matchbox/src/components/ThemeProvider/globalStyles.js]: We should, but the JS tokens don't export a default font size, just the scale. This is one token we should add though.

Comment (jonambas) [reply packages/matchbox/src/components/ThemeProvider/globalStyles.js]: I'll cut a ticket to add a new token

**Pull request merged**